include/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -283,6 +282,10 @@ SECTIONS
 	}
 #endif
 
+#if defined(CONFIG_RAM_FUNCTION)
```

Comment:
@SebastianBoe could you look at this, once more?
It looks like this section is, now, the very first section; so now _image_ram_start will not be the start of SRAM. Should this be OK, or requires a fix?